how to contribute part added to redme.md file (#24)
grafana/k6-learn: README.md

- [Marie Cruz](https://github.com/mdcruz)
- [Matt Dodson](https://github.com/MattDodsonEnglish)
- [Nicole van der Hoeven](https://github.com/nicolevanderhoeven)
- [Paul Balogh](https://github.com/javaducky)

## How to contribute?

1. If there an issue does not exist, start by creating it under https://github.com/grafana/k6-learn/issues.
2. [Fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks) this [repository](https://github.com/grafana/k6-learn). 
3. Make the changes in your forked repository. Note that new branches, changes and pushes will only affect your repository.

> 4. Once ready, create a [Pull Request fro your fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork), mentioning the issue it solves (step 1). Keep in mind you might need to [sync it](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork).

Author: ```suggestion
4. Once ready, create a [Pull Request from your fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork), mentioning the issue it solves (step 1). Keep in mind you might need to [sync it](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork).
```

> 5. Once you have the create the Pull Request, your changes now will be reviewed and either accepted or you will be asked for modifications.

Author: ```suggestion
5. Once you have created the Pull Request, your changes now will be reviewed and either accepted or you will be asked for modifications.
```

- [Marie Cruz](https://github.com/mdcruz)
- [Matt Dodson](https://github.com/MattDodsonEnglish)
- [Nicole van der Hoeven](https://github.com/nicolevanderhoeven)
- [Paul Balogh](https://github.com/javaducky)

## How to contribute?

> First, go to the main page: https://github.com/grafana/k6-learn. Fork the repository by clicking the "Fork" button. You should be redirected to your github /username/k6-learn. Now if everything is ok, you can clone the repository. You can do it for example by using your console with "git clone https://github.com/username/k6-learn". Nwo, add the upstream for your repo: "git remote add upstream https://github.com/grafana/k6-learn". Note that new branches, changes and pushes will only affect your repository. If you want to create a pull request go to /username/k6-learn on github and in the "Pull requests" tab create a pull request. At this point you have to decide from which branch you want to create a new pull request. Your changes now will be reviewed and either accepted or you will be asked for modifications.

Reviewer: Thanks for writing this! 

I would use a bullet list, as it's clearer. 

And I'd suggest not mention to commands that users should know and can change depending on how you work. Or if GitHub changes in the future. E.g. for minor typos or changing a single markdown page I usually do that on GitHub without cloning locally. If we want to explain a bit how to collaborate with forks, we can probably add some links to GitHub docs (and keep and eye if they change urls/content). My suggestion below:

```suggestion
1. If there an issue does not exist, start by creating it under https://github.com/grafana/k6-learn/issues.
2. [Fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks) this [repository](https://github.com/grafana/k6-learn). 
3. Make the changes in your forked repository. Note that new branches, changes and pushes will only affect your repository.
4. Once ready, create a [Pull Request fro your fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork), mentioning the issue it solves (step 1). Keep in mind you might need to [sync it](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork).
5. Once you have the create the Pull Request, your changes now will be reviewed and either accepted or you will be asked for modifications.
```
